### PR TITLE
Openstack compare copy update

### PR DESCRIPTION
**Pull request by jpmartinspt**
## Done

Rebuild the page as specified in https://github.com/canonical-web-and-design/web-squad/issues/3939

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

Fixes https://github.com/canonical-web-and-design/web-squad/issues/3939

### CODE REVIEW

**Comment by anasereijo** [comment]
@jpmartinspt thanks for doing this. 
In my opinion, this needs design time (and UX too probably). @SirSamTumless could you allocate some time for the next iteration?

**Comment by cristinadresch** [comment]
@jpmartinspt is this ready for code review?

**Comment by SirSamTumless** [comment]
@carkod Why's this blocked?

**Comment by anasereijo** [comment]
@SirSamTumless as discussed, here are my thoughts:

If we keep adding stuff to this table we need to rethink the whole thing as it is very confusing now and hard to go through the info.  We are now showing a mix of things such as expanding text, icons, $$ and ratings.

The purpose of using an expanding table was so the user could compare easily and quickly compare the 3 companies and then read a bit more about Canonical (by expanding the rows). I don't think we are serving that purpose anymore. 

If we really need to keep all these things I think we will need UX help.

**Comment by clagom** [comment]
I agree with @anasereijo, this needs UX too: with the new elements in there I don't think the accordion pattern works as well. Thanks for flagging this Ana.
@SirSamTumless Do we still need the comparison at all? Do we have a brief for the changes? We need more context.

**Comment by SirSamTumless** [comment]
@tytus-kurek The team have looked at this (see the demo link) and with the rating, it really doesn't work. I think this needs a rethink. Also, I personally don't think the ratings are great - we've given ourselves full ratings for everything which screams inauthenticity to me.

**Comment by tytus-kurek** [comment]
@SirSamTumless what is the exact problem with the new layout?

Is it that we use ratings? I discussed that with Alex some time ago and we are already using ratings at ubuntu.com/kubernetes/compare which was another iteration of the "compare" pages, so the goal was to standardise on a common framework.

Is it that we gave ourselves 5/5 everywhere? Again, we've done the same at ubuntu.com/kubernetes/compare and it passed the UX review. Also, this is not much different from what everyone is doing. Have a look at https://platform9.com/openstack-distributions-comparison/ , for example. They undermine any other OpenStack distributions while highlighting the benefits of Platform9 Managed OpenStack.

Is it that we use "+" next to the ratings? We can get rid of it if needed.

To sum up, I'm happy to adjust to a company-wide standard for the "compare" pages creation, but I'd like to make sure I'm not the only one who's tasked to do that.

**Comment by clagom** [comment]
@tytus-kurek I think the main point is the readability of the table. If you put side by side our table with the link you shared I think the difference is very prominent. Our table has price ($$$), check icons, rating (of what exactly?) and + to open up more information. The link you shared compares just a X or a icon/logo of the product. I think we just need to work on the interaction (do we need "+" at all? more info at all?) and to simplify the content if we can? As it is, makes users wonder what is going on and takes time to engage and interpret what we want to say.
In short, I think we would like to improve it.
CC @SirSamTumless @anasereijo

**Comment by tytus-kurek** [comment]
@SirSamTumless Let me suggest something. Since you already have a demo of the page with all the recent changes, why don't we update the parts of the page you're happy with (the title, the top paragraph, other paragraphs, table structure, etc.) and continue working on the exact layout of the comparison table in parallel. I'd be happy to adjust to whatever you come up with when updating K8s pages. Would that work for you?

**Comment by tytus-kurek** [comment]
@clagom ^^ sorry, I didn't realize the latest comment came from you.

**Comment by jpmartinspt** [comment]
@clagom @anasereijo @SirSamTumless  @tytus-kurek I have removed most of the things we don't like about that table. are we good to merge it as it stands now and think about a better layout on a follow-up PR?

**Comment by tytus-kurek** [comment]
@jpmartinspt LGTM and I'm happy to continue working on a better layout in the future.

**Comment by anasereijo** [comment]
@jpmartinspt LGTM and yes, we should look into this in the near future.

I have some suggestions:

- [ ] row elements are not vertically aligned.
Screenshot 2021-04-07 at 17 06 18

- [ ] Quote: can you make it look like this example? You can check the code [here](https://ubuntu.com/0_docs/content_strips) (scroll down)

Screenshot 2021-04-07 at 17 10 25

- [ ] after changing the quote styling make the following section (More economical than Red Hat...) white background and the next one light-grey and so on.

**Comment by clagom** [comment]
Thanks @jpmartinspt ;)

**Comment by jpmartinspt** [comment]
@anasereijo updated!

**Comment by anasereijo** [comment]
lgtm 👍 thanks`!

File: templates/openstack/compare.html [hunk 1]
```diff
@@ -80,183 +81,234 @@ <h1>OpenStack distro comparison</h1>
...
 
         <tr>
-          <th>Supported hypervisors</th>
-          <td class="p-accordion is-compact">
-            <button class="p-accordion__tab" id="tab-hypervisors" role="tab" aria-controls="content-hypervisors" aria-expanded="false">KVM</button>
+          <th>Hypervisors</th>
+          <td class="p-accordion is-compact ">
+            <button class="p-accordion__tab " id="tab-hypervisors" role="tab" aria-controls="content-hypervisors" aria-expanded="false">KVM</button>
             <div class="p-accordion__panel" aria-hidden="true" id="content-hypervisors">
               <p>Charmed OpenStack provides support for KVM hypervisors. Other hypervisor out of the box are available through the consulting services.</p>
```
Comment (anthonydillon) [inline templates/openstack/compare.html:222]: This content is different in the copy doc

File: templates/openstack/compare.html [hunk 2]
```diff
@@ -290,18 +342,19 @@ <h2>Cheaper than Red Hat OpenStack Platform</h2>
       }}
     </div>
     <div class="col-8">
-      <h2>Simpler than Mirantis Cloud Platform</h2>
-      <p>Instead of relying on various interconnected tools, Canonical uses <a href="https://docs.openstack.org/charm-guide/latest/">OpenStack Charms</a> for OpenStack deployments, operations and upgrades while providing the same value. OpenStack Charms are orchestrated by Juju which abstracts the entire OpenStack complexity, enables an IaC (infrastructure as code) approach and provides a SaaS (software as a service) experience. This results in a simpler OpenStack platform, fewer resources required to maintain it and reduced operational costs.</p>
+      <h2>More straightforward than Mirantis Cloud Platform</h2>
+      <p>Instead of relying on random interconnected tools, Canonical leverages the concept of operators for OpenStack deployments, operations and upgrades. The OpenStack Charmed Operators (OpenStack Charms) enable model-driven architecture and abstract the entire OpenStack complexity. This results in a simpler OpenStack platform, fewer resources required to maintain it and reduced operational costs.</p>
```
Comment (anthonydillon) [inline templates/openstack/compare.html:346]: OpenStack Charms should be a link

File: templates/openstack/compare.html [hunk 3]
```diff
@@ -290,18 +342,19 @@ <h2>Cheaper than Red Hat OpenStack Platform</h2>
...
+      <p><a href="#">Learn more &nbsp;&rsaquo;</a></p>
     </div>
   </div>
 </section>
 
-<section class="p-strip--light">
+<section class="p-strip">
   <div class="row u-equal-height">
     <div class="col-8">
       <h2>Open source, unlike VMware</h2>
```
Comment (anthonydillon) [inline templates/openstack/compare.html:355]: This one doesn't have a comma in the copy doc

File: templates/openstack/compare.html [hunk 4]
```diff
@@ -290,18 +342,19 @@ <h2>Cheaper than Red Hat OpenStack Platform</h2>
...
-      <p>Charmed OpenStack is fully open source which allows it to benefit from a broader community, faster development process and broader selection of technology choices than those available in VMware's vRealize platform. Choosing an open source platform avoids vendor lock-in. Charmed OpenStack is also much cheaper than VMware vRealize. Under certain circumstances, migrating from VMware to Charmed OpenStack can reduce costs by 88%.</p>
-      <p><a href="/engage/vmware-to-charmed-openstack">Read the whitepaper &mdash; "From VMware to Charmed OpenStack"&nbsp;&rsaquo;</a></p>
+      <p>Canonical's Charmed OpenStack is fully open source which allows it to benefit from a broader community, faster development process and wider selection of technology choices than those available in VMware's platform. Choosing an open source platform avoids vendor lock-in and significantly reduces infrastructure costs.</p>
```
Comment (anthonydillon) [inline templates/openstack/compare.html:356]: This one has different content in the copy doc

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some comments in the code and its missing the last section "More cost-efficient than public clouds" from the copy doc.

File: templates/openstack/compare.html [hunk 5]
```diff
@@ -319,6 +372,37 @@ <h2>Open source, unlike VMware</h2>
...
+        width="180",
+        height="180",
+        hi_def=True,
+        loading="auto",
+        ) | safe
+      }}
+    </div>
+    <div class="col-8">
+      <h2>Contact us</h2>
+      <p>Let our private cloud team help with:</p>
```
Comment (anthonydillon) [inline templates/openstack/compare.html:391]: This section has suggestions in the copy doc

**Comment by jpmartinspt** [comment]
@anthonydillon everything should be fixed now.

File: templates/openstack/compare.html [hunk 6]
```diff
@@ -80,183 +81,234 @@ <h1>OpenStack distro comparison</h1>
...
 
         <tr>
           <th>Bare-metal provisioning tool</th>
-          <td class="p-accordion is-compact">
-            <button class="p-accordion__tab" id="tab-bare-metal-tool" role="tab" aria-controls="content-bare-metal-tool" aria-expanded="false">MAAS</button>
+          <td class="p-accordion is-compact ">
+            <button class="p-accordion__tab " id="tab-bare-metal-tool" role="tab" aria-controls="content-bare-metal-tool" aria-expanded="false">MAAS</button>
             <div class="p-accordion__panel" aria-hidden="true" id="content-bare-metal-tool">
-              <p><a href="https://maas.io">MAAS</a> is a bare-metal provisioning tool developed and maintained by Canonical. It is used by Charmed OpenStack, Mirantis Cloud Platform and in the OpenStack Airship project.</p>
+              <p><a href="https://maas.io">MAAS</a> is a bare-metal provisioning tool that turns a data centre into a true bare metal cloud.</p>
```
Comment (anthonydillon) [inline templates/openstack/compare.html:190]: All the external links should have a `p-link--external` class on them.

**Review by anthonydillon — APPROVED** [review]
LGTM with a small request

**Comment by jpmartinspt** [comment]
@anthonydillon sorry I missed the requests, I'll open another PR tomorrow.
